Content.Shared/_RMC14/Xenonids/Fortify/XenoFortifyComponent.cs

```diff
@@ -23,7 +23,7 @@ public sealed partial class XenoFortifyComponent : Component
     public float ExplosionMultiplier = 0.4f;
 
     [DataField, AutoNetworkedField]
-    public string[] ImmuneToStatuses = { "Stun", "KnockedDown" };
+    public string ImmuneToStatus = "KnockedDown";
```

Comment:
probably should keep this as a list incase someone wants to add more effects in the future, it's easier to implement